knative-operator/pkg/controller/knativeserving/knativeserving_controller_test.go

```diff
@@ -124,6 +125,7 @@ func TestExtraResourcesReconcile(t *testing.T) {
 			ns := &dashboardNamespace
 
 			cl := fake.NewClientBuilder().WithObjects(ks, ingress, ns, &servingNamespace).Build()
+			consoleclidownload.ConsoleInstalled.Store(true)
```

Comment:
We should adjust the test to simulate no console. Then console should be enabled, after a while (by go routine). In that way, we could test proper waiting for the console to became available.

Reply:
Ok we can try that but I was expecting a e2e test for this anyway.

Reply:
Yeah. An e2e test would be better, I think.

Reply:
I am thinking about it and it is not that easy either. Have to run without console and then apply it. We need a separate ci flow for this since we can just uninstall console.